### PR TITLE
(wip) Release note for 0.10.0

### PR DESCRIPTION
**Pull request by eed3si9n**
Starting a draft release note. Feel free to push revisions on on this branch.

### CODE REVIEW

File: notes/0.10.0.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    person: Person = Person(foo,20)
+
+The `Defaults` mode automatically derives `Pickler[Person]` from the primitive picklers at compile-time!
+Because the code is statically generated, we can inline the string manipulations and make it fast.
+([Faster than Java serialization or Kryo][Miller], which also does not require schema)
+
+Note, because `Pickler[A]` is a typeclass, Pickling can be retrofitted to `Person`
+without modifying the class to inherit [Serializable][1] or something like that.
+
+## DYI protocol stack
```
Comment (jsuereth) [inline notes/0.10.0.markdown]: s/DYI/DIY/

File: README.md [hunk 2]
```diff
@@ -7,19 +7,28 @@ scala/pickling
 
 **Scala Pickling** is an automatic serialization framework made for Scala. It's fast, boilerplate-free, and allows users to easily swap in/out different serialization formats (such as binary, or JSON), or even to provide their own custom serialization format.
 
-### Basic usage (0.9.0)
+### Defaults mode (0.10.0)
 
 ```scala
-import scala.pickling._, json._
+scala> import scala.pickling.Defaults._, scala.pickling.json._
```
Comment (jsuereth) [inline README.md:13]: Defaults is still needed here?

**Comment by jsuereth** [comment]
Nice kick-off of the release notes!

File: README.md (hunk 2 above)
Comment (eed3si9n) [inline README.md:13]: Yes. We need this because pickling's package object is implicit-free.

File: README.md [hunk 3]
```diff
@@ -114,30 +123,42 @@ If you're a library author, you can provide the convenience object as your proto
...
+     |                // Don't fall back to runtime picklers
+     |                implicit val so = static.StaticOnly
+     |                // Provide custom functions
+     |                def toJsonString[A: Pickler](a: A): String =
+     |                  functions.pickle(a).value
+     |                def fromJsonString[A: Unpickler](s: String): A =
+     |                  functions.unpickle[A](json.JSONPickle(s))
+     |              }
+     |            }
+appleProtocol: scala.pickling.pickler.PrimitivePicklers with scala.pickling.pickler.RefPicklers with scala.pickling.json.JsonFormats{implicit val applePickler: scala.pickling.Pickler[Apple] with scala.pickling.Unpickler[Apple] with scala.pickling.Generated; implicit val so: scala.pickling.static.StaticOnly.type; def toJsonString[A](a: A)(implicit evidence$1: scala.pickling.Pickler[A]): String; def fromJsonString[A](s: String)(implicit evidence$2: scala.pickling.Unpickler[A]): A} = $anon$1@2b033c35
```
Comment (jsuereth) [inline README.md:144]: that type....... it HURTS

**Comment by jsuereth** [comment]
LGTM

File: README.md (hunk 3 above)
Comment (eed3si9n) [inline README.md:144]: I agree it should be a trait.

**Pull request merged**